mitodl/mitxonline
# Add courses app

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #3 

#### What's this PR do?
Add the `courses` app

#### How should this be manually tested?
- Login to /admin, you should see many admin sections for the courses app related to courses, course runs, programs, enrollments, and grades. 
- Create some programs, courses, and course runs
- Verify that live programs and courses show up at the apis `/api/programs/` and `/api/courses/`.
- Verify that Course Runs show up at `/api/course_runs`, this API does not filter on `live=True`

#### Any background context you want to provide?
This pulls is more than we probably need in the short term, but we'll need it all within a few months so I've chosen to bring them in now so we don't have to expend a lot of effort dealing with merging things later on.
